.devcontainer/Dockerfile

```
@@ -0,0 +1,15 @@
+
+ARG VARIANT=16-bullseye
+FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:${VARIANT}
```

When I set it up, this line is slightly different. I'll do a suggestion here just so you can see the diff:

```suggestion
FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:0-${VARIANT}
```

Any idea what the `0-` means?

---

The `0-` has to do with the docker image semantic version that Microsoft is doing. Based on the documentation here, it would be better practice now to keep the `0-` in. I originally removed it because older contributions didn't use the versioning but I now realize it's better to keep it now.